tests/test_version.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf8 -*-
+
+# TODO: This is a demo of the way we can report he cluster version.
+# Code here will be included in some python module when the current changes
+# settles down. Also for this report to be actually useful, we need to run this
+# as autouse fixture (again, exact way to do this needs to be aligned with
+# others, eg. https://github.com/red-hat-storage/ocs-ci/pull/173).
+
+
+import argparse
+import logging
+import os.path
+import sys
+
+from ocs.ocp import OCP
+import ocsci.config
+
+# TODO will require a change when helpers module will be moved somewhere else
+# later
+from .helpers import create_unique_resource_name
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: convert this to a fixture later, align with setup, teardown and testing
+# @pytest.fixture(scope="session", autouse=True)
+def test_version():
+
+    # note: besides reporting the version in the logs, we also create a version
+    # file in 'cluster_path' directory, which can copypaste into bugzilla as a
+    # base of "version of ocs components" section of a bugreport.
+    version_filename = os.path.join(
+        ocsci.config.ENV_DATA['cluster_path'],
+        create_unique_resource_name("cluster", "version"))
```

Comment:
Here I would propose to use full timestamp - e.g. `2019-06-11T15-20-58` or `20190611T152058` or something similar... from which will be immediately clear the exact time... (the output from `create_unique_resource_name` starts with day, without month and year, which is quite unclear without examining the source code...)

Reply:
Ack, I will use that instead of `create_unique_resource_name`.

Reply:
done in 6a1e958